frontend/app/schemas/application-history-service-schemas.server.ts

```diff
@@ -0,0 +1,149 @@
+import { z } from 'zod';
+
+export const applicationSchema = z.object({
```

Comment:
I don't see `applicationSchema` being used anywhere outside this file. If that's the case, we can remove the `export`.

Same comment for `ApplicationList` type at the bottom of this file.

Reply:
Fixed:  a3363de42a431757737fa4c338f8126b2aca30e5